dsmr_parser/objects.py

```diff
@@ -27,9 +28,13 @@ def __init__(self, *args, **kwargs):
         self._mbus_devices = []
         super().__init__(*args, **kwargs)
 
-    def add(self, obis_reference, dsmr_object):
-        # Update name mapping used to get value by attribute. Example: telegram.P1_MESSAGE_HEADER
-        obis_name = obis_name_mapping.EN[obis_reference]
+    def add(self, obis_reference, dsmr_object, obis_name=None):
+        # Update name mapping used to get value by attribute. Example: telegram.P1_MESSAGE_HEADER.
+		# For backwards compatibility
+        if obis_name is not None:
+            obis_name = obis_name_mapping.EN[obis_name]
+        else:
+            obis_name = obis_reference
```

Comment:
Ideally you want to prevent this if statement and have the translation be done for all DSMR specifications. This can be seen as technical debt

But maybe we can use general names as mentioned in my previous comment and prevent this issue all together?